apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/controllers/trigger_policy_controller.ex

```diff
@@ -51,7 +51,8 @@ defmodule Astarte.RealmManagement.APIWeb.TriggerPolicyController do
 
   def show(conn, %{"realm_name" => realm_name, "id" => id}) do
     with {:ok, policy_source} <- Policies.get_trigger_policy_source(realm_name, id),
-         {:ok, decoded_json} <- Jason.decode(policy_source) do
+         # Use (safe) atoms as keys to simplify handler normalization in Trigger Policy View
```

Comment:
This is ok as a temporary workaround, but in the long run we should add a `from_source` function to `Astarte.Core.Trigger.Policy`. So please add another line with:
```
# TODO: move this to a function in Astarte Core building a Policy from its source
```
so we don't forget to do this in a following iteration